src/sentry/static/sentry/app/views/dashboardsV2/controls.tsx

```diff
@@ -86,17 +86,6 @@ class Controls extends React.Component<Props> {
       return (
         <ButtonBar gap={1} key="create-controls">
           {cancelButton}
-          <Button
-            data-test-id="dashboard-commit"
-            onClick={e => {
-              e.preventDefault();
-              onCommit();
-            }}
-            priority="primary"
-            icon={<IconAdd size="xs" isCircled />}
-          >
-            {t('Create Dashboard')}
-          </Button>
```

Comment:
Without this button how does the dashboard get saved? Because of the return we don't get to line 130.

Reply:
OOOHHH I misunderstood this - it's not "Create Dashboard", it's "save" or "finish"... got it. I'll fix it.

Reply:
it also means "create this dashboard"